lib/routes/qiche365/recall.ts

```diff
@@ -0,0 +1,58 @@
+import { Route, Data, DataItem } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import timezone from '@/utils/timezone';
+
+const baseUrl = 'https://www.qiche365.org.cn';
+
+export const route: Route = {
+    path: '/recall/:channel',
+    name: '汽车召回',
+    example: '/qiche365/recall/1',
+    parameters: { channel: '频道，见下表' },
+    description: `| 国内召回新闻 | 国内召回公告 | 国外召回新闻 | 国外召回公告 |
+  | ------------ | ------------ | ------------ | ------------ |
+  | 1            | 2            | 3            | 4            |`,
+    categories: ['government'],
+    maintainers: ['huanfe1'],
+    handler,
+    url: 'qiche365.org.cn/index/recall/index.html',
+};
+
+async function handler(ctx): Promise<Data> {
+    const { channel } = ctx.req.param();
+    const link = `https://www.qiche365.org.cn/index/recall/index/item/${channel}.html?loadmore=1`;
+
+    const html = await cache.tryGet(link, async () => {
+        const { html } = await ofetch(link, {
+            method: 'get',
+            headers: {
+                'Accept-Language': 'zh-CN,zh;q=0.9',
+            },
+        });
+        return html;
+    });
```

Comment:
Don't cache this request. If you really want to cache it, please set the response with `tryGet` 3rd parameter of  [`tryGet`](https://docs.rsshub.app/joinus/advanced/use-cache) to `config.cache.routeExpire` and 4th to `false`